Automattic/wp-calypso
# Offer Reset: create the Jetpack Pricing page

#### Changes proposed in this Pull Request

* Create the new pricing page in `cloud.jetpack.com/pricing` based on the new plans grid.

#### Tasks not included in this PR

* Creating a master bar that looks like the one in jetpack.com/pricing.
* Making CTAs that point to the checkout step work. 

#### Implementation notes

* Fixed glitch in the `PlansFilterBar` that happened when the bar went from sticky to normal and the reverse.

#### Testing instructions

* Run this PR with `yarn start-jetpack-cloud`. 
* Go through all the steps being authenticated and then in an incognito window (unauthenticated).
* Visit `http://jetpack.cloud.localhost:3000/pricing`.
* Verify that sure there are no errors in the console.
* Verify that sure you can move between cards (you might not be able to purchase).
* Verify that there is no noticeable glitch/jump when the `PlansFilterBar` goes from normal to sticky.
* Verify that nothing has changed in Calypso's Plans page.

Fixes 1169247016322522-as-1189391103512220

#### Demo
![image](https://user-images.githubusercontent.com/3418513/91758213-00476980-eba6-11ea-9530-d4495b7a93b3.png)
